src/v2/Apps/Fair/Utils/useExhibitorsTabOffset.ts

```diff
@@ -0,0 +1,17 @@
+import { themeProps } from "@artsy/palette"
+import { useNavBarHeight } from "v2/Components/NavBar/useNavBarHeight"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+export const useExhibitorsTabOffset = () => {
+  const {
+    height: [mobileNavBarHeight, desktopNavBarHeight],
+  } = useNavBarHeight()
+
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+  const stickyTabsHeight = 150
+
+  const offset =
+    (isMobile ? mobileNavBarHeight : desktopNavBarHeight) + stickyTabsHeight
+
+  return offset
```

Comment:
I guess we should just write a utility function to detect the sticky height and handle all this automatically since this has come up a few times.

Reply:
I think we should create a new ticket for this, do you?

Reply:
👍